test/e2e/case13_templatization_test.go

```diff
@@ -540,7 +540,7 @@ var _ = Describe("Test templatization", Ordered, func() {
 			Expect(configMap2.ObjectMeta.Labels).To(HaveKeyWithValue("namespace", "default"))
 		})
 
-		It("Should fail when the namespace doesn't match after template resolution", func(ctx SpecContext) {
+		It("Should fail when the namespace doesn't match after template resolution", func(_ SpecContext) {
```

Comment:
If it's not being used, it can just be removed here.

```suggestion
		It("Should fail when the namespace doesn't match after template resolution", func() {
```